### PR TITLE
{data,vis}[intel/2015b] scikit-image 0.11.3, scikit-learn 0.16.1, Pillow 2.9.0

### PR DESCRIPTION
**Pull request by boegel**
requires ~~#1975~~ (for Qhull patch)

### CODE REVIEW

**Comment by boegel** [comment]
@wpoely86: please review?

File: easybuild/easyconfigs/c/CMake/CMake-3.2.1-GNU-4.9.3-2.25.eb [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+homepage = 'http://www.cmake.org'
+description = """CMake, the cross-platform, open-source build system.
+ CMake is a family of tools designed to build, test and package software."""
+
+toolchain = {'name': 'GNU', 'version': '4.9.3-2.25'}
+
+source_urls = ['http://www.cmake.org/files/v%(version_major_minor)s']
+sources = [SOURCELOWER_TAR_GZ]
+
+dependencies = [('ncurses', '5.9')]
```
Comment (wpoely86) [inline easybuild/easyconfigs/c/CMake/CMake-3.2.1-GNU-4.9.3-2.25.eb]: why not go to `6.0`? I thing I pushed it somewhere already.

**Comment by wpoely86** [comment]
looks fine

File: easybuild/easyconfigs/c/CMake/CMake-3.2.1-GNU-4.9.3-2.25.eb (hunk 1 above)
Comment (boegel) [inline easybuild/easyconfigs/c/CMake/CMake-3.2.1-GNU-4.9.3-2.25.eb]: I haven't seen a PR for it, and if we go with ncurses 6.0, we need to do it everywhere (e.g. also in libreadline, etc.).

**Comment by boegel** [comment]
Jenkins: test this please

File: easybuild/easyconfigs/c/CMake/CMake-3.2.1-GNU-4.9.3-2.25.eb (hunk 1 above)
Comment (wpoely86) [inline easybuild/easyconfigs/c/CMake/CMake-3.2.1-GNU-4.9.3-2.25.eb]: OKay, then it is in my unpushed Clang 3.7 branch. I will try to push it out ASAP. It uses ncurses 6 for everything (libreadline etc).

**Comment by boegel** [comment]
Test report by @boegel
**FAILED**
Build succeeded for 6 out of 7 (7 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/29cc5576c5368c7997c5 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 7 out of 7 (7 easyconfigs in this PR)
Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/54a46ecbaab6026bb6bc for a full test report.

**Comment by boegel** [comment]
Thanks for the review @wpoely86!

**Pull request merged**